crates/humanode-runtime/src/lib.rs

```diff
@@ -1351,6 +1338,10 @@ impl_runtime_apis! {
                 access_list.unwrap_or_default(),
                 is_transactional,
                 validate,
+                // TODO: should we define limit for weight and transaction len.
+                // <https://github.com/paritytech/frontier/pull/1039>.
```

Comment:
Let's create our own issue for this

Reply:
https://github.com/humanode-network/humanode/pull/848/commits/8d66bedd177b222cba57ee2bc4adcac82fd14061

https://github.com/humanode-network/humanode/issues/864